src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/TagletWriterImpl.java

```diff
@@ -141,6 +143,9 @@ Context within(DocTree tree) {
     private final Contents contents;
     private final Context context;
 
+    // Threshold for length of @see tag label for switching from inline to block layout.
+    private static final int SEE_TAG_MAX_INLINE_LENGTH = 30;
+
```

Comment:
For future consideration, we should look at encoding numbers like this in a config file.